Cargo.toml

```diff
@@ -14,18 +14,17 @@ members = [
   "provider-example",
   # the main library and tests
   "rustls",
-  # experimental post-quantum algorithm support
-  "rustls-post-quantum",
   # rustls cryptography provider integration tests
   "rustls-provider-test",
 ]
+exclude = [
```

Comment:
It would otherwise be included because of `bogo`'s path dependency on it. See https://doc.rust-lang.org/cargo/reference/workspaces.html#the-members-and-exclude-fields